homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -197,7 +194,7 @@ def volume_level(self):
             self._source_list = await fs_device.get_mode_list()
 
         if not self._volume_steps:
-            self._volume_steps = await fs_device.get_volume_steps()
+            self._volume_steps = int(await fs_device.get_volume_steps()) - 1
```

Comment:
The API seems to include 'zero' in the number of steps (e.g. if the range is 0-40 then get_volume_steps returns 41), subtract one to get the max volume step.

Reply:
Thanks for this information, now your implementation makes sense 👍

Reply:
Please add that comment though ✌